apps/storybook/.storybook/main.ts

```diff
@@ -85,6 +89,8 @@ module.exports = {
   },
```

Comment:
what's you opinion on autodocs ?

Reply:
IMO not needed for Cal.com since all stories have an explicit `mdx`

Reply:
I didn't see any difference when adding or omitting this option so I don't think it matters.

What was your reasoning behind lazy builds?